specification/2.0/schema/animation.channel.target.schema.json

```diff
@@ -10,7 +10,7 @@
             "description": "The index of the node to target."
         },
         "path": {
-            "description": "The name of the node's TRS property to modify, or the \"weights\" of the Morph Targets it instantiates. For the \"translation\" property, the values that are provided by the sampler are the translation along the x, y, and z axes. For the \"rotation\" property, the values are a quaternion in the order (x, y, z, w), where w is the scalar. For the \"scale\" property, the values are the scaling factors along the x, y, and z axes.",
+            "description": "The name of the node's TRS property to modify, or the `\"weights\"` of the Morph Targets it instantiates. For the `\"translation\"` property, the values that are provided by the sampler are the translation along the X, Y, and Z axes. For the `\"rotation\"` property, the values are a quaternion in the order (x, y, z, w), where w is the scalar. For the `\"scale\"` property, the values are the scaling factors along the X, Y, and Z axes.",
```

Comment:
I don't recall if we are still using backtics for code-quoting in any other JSON schema descriptions.  When I was first working on the VSCode extension, these descriptions were processed as plain text, not markdown.  I don't think that's changed, but I haven't checked in some years since then.  If it works in the wetzel output though, that might be all that's needed.

Reply:
That is needed for the wetzel output to be consistent. We can look into VSCode UI later.